handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -405,9 +414,19 @@ protected static X509TrustManager chooseTrustManager(TrustManager[] managers) {
         throw new IllegalStateException("no X509TrustManager found");
     }
 
+    protected static X509KeyManager chooseX509KeyManager(KeyManager[] kms) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)